<!DOCTYPE html>
<html>
<head>
<title>GTEx eVNTR</title>
</head>
<body>

<h1>Description</h1>
<p>Expression VNTR (eVNTR) is defined by association between repeat length of VNTR and level of expression of genes within +/-500 kb of each other. The gene expression levels were obtained from RNAseq and the repeat length for VNTRs were estimated using CNVnator on Whole Genome Sequencing.
</p>

<h1>Display Convention</h1>
<p>The Track shows association between VNTR and gene expression in each of the 48 Tissues. Each Tissue is colored using GTEx-convention</p>

<h1>Method</h1>
<p>We obtained Illumina 150bp paired-end WGS data and resulting variant calls made using GATK in 620 individuals from The Genotype-Tissue Expression (GTEx) project from dbGAP (Accession ID phs000424.v7.p2). RNAseq data for these samples were downloaded from GTEX portal (Version 7, https://www.gtexportal.org/), comprising quality-controlled and processed files for 48 tissues generated by the GTEx Consortium. These data were aligned to reference genome, had already undergone filtering to remove genes with low expression, and been subject to rank based inverse normal transformation.
</p>

<p>
Using RNAseq data from the filtered set of 404 WGS samples that passed our quality control steps, gene expression data were adjusted for gender, RNAseq platform, the first three principal components from SNV genotypes, and between 15-60 covariates per tissue estimated using PEER. Within each tissue, we performed linear regression between VNTR copy number and
corrected expression level of each gene located within Â±500kb using the lm function in R. We applied a False Discovery Rate correction, and reported all VNTR:gene pairs with FDR q<0.1 in any tissue. 
</p>

<h1>References</h1>
<p>The manuscript is currently under review. A copy of the manuscript is available here: <a href="https://www.biorxiv.org/content/10.1101/2020.12.16.423078v1">https://www.biorxiv.org/content/10.1101/2020.12.16.423078v1</a></p>
